spec/models/charge_spec.rb

```diff
@@ -1,4 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Charge, :type => :model do
+  it{should validate_presence_of :token}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.